### ISSUE TITLE
help me 

### ISSUE THREAD

**Issue by ghost** [i-make-robots/truchetTiles#1]
hello I have a problem at this level I downloaded your file and I started the script and it gives me this problem
![Sans titre](https://user-images.githubusercontent.com/78557886/123450758-57a74200-d5dd-11eb-9914-ef2d94cde773.png)

**Comment by i-make-robots** [comment]
Error says it can’t load the jpg file. Is your copy missing?

**Comment by ghost** [comment]
I haven't touched anything in your folder it's all your files but it marks the error
![Sans titre](https://user-images.githubusercontent.com/78557886/123459289-621a0980-d5e6-11eb-8be8-9694bccf9d64.png)

**Comment by i-make-robots** [comment]
What is the entire error message? All the red text.

**Comment by ghost** [comment]
The file "alan-turing.jpg" is missing or inaccessible, make sure the URL is valid or that the file has been added to your sketch and is readable.
The file "code.jpg" is missing or inaccessible, make sure the URL is valid or that the file has been added to your sketch and is readable.
NullPointerException

**Comment by i-make-robots** [comment]
I don’t know.  All the files are clearly there.

**Comment by ghost** [comment]
yes it says that the file is not there and that there is a problem with the command img.filter (GRAY); when you run it does it work fine?

**Comment by i-make-robots** [comment]
Yes, it’s fine here. 
Never look at the last error. Look at the first. Img is not loaded because it can’t find the file. Your app is running in the wrong directory somehow.
